### PR TITLE
[WIP][16.0][IMP] account_reconcile_oca: Create exchange diff move when concile c…

### PR DESCRIPTION
**Pull request by aritzolea**
…urrencies don't match

### CODE REVIEW

File: account_reconcile_oca/models/account_partial_reconcile.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+class AccountPartialReconcile(models.Model):
+
+    _inherit = "account.partial.reconcile"
+
+    @api.model_create_multi
+    def create(self, vals_list):
+        if self.env.context.get("no_foreign_currency", False):
+            for vals in vals_list:
+                vals["debit_amount_currency"] = 0
```
Comment (etobella) [inline account_reconcile_oca/models/account_partial_reconcile.py:17]: Why not credit? It seems something weird to do it with this context. I would recommend to give some more context to your issue, how it was raised and so on...

**Comment by etobella** [comment]
After reviewing Odoo Code for reconcile I do not agree in this approach. this fields are computed automatically on reconcile function....

Can you explain the reason of this PR?

**Comment by SoniaViciana** [comment]
I haven't tried the PR yet, but want to solve this:

image

image

**Review by etobella — CHANGES_REQUESTED** [review]
But removing the logic of foreign currency should not be an option. 

There is probably a bug in the code, but applying this patch is not the solution

**Comment by aritzolea** [comment]
Closing PR after checking the bug is coming indeed from Core Odoo 16. We are trying to get a solution but applied to core directly.

**Comment by etobella** [comment]
Thanks @aritzolea for checking

**Pull request closed without merging**